src/ado2gh/Commands/RewirePipelineCommand.cs

```diff
@@ -72,8 +72,8 @@ public async Task Invoke(string adoOrg, string adoTeamProject, string adoPipelin
             var ado = _lazyAdoApi.Value;
 
             var adoPipelineId = await ado.GetPipelineId(adoOrg, adoTeamProject, adoPipeline);
-            var pipelineDetails = await ado.GetPipeline(adoOrg, adoTeamProject, adoPipelineId);
-            await ado.ChangePipelineRepo(pipelineDetails, githubOrg, githubRepo, serviceConnectionId);
+            var (defaultBranch, clean, checkoutSubmodules) = await ado.GetPipeline(adoOrg, adoTeamProject, adoPipelineId);
```

Comment:
here it's ok to use camel cased properties in a value tuple because as I mentioned there are going to be uses as local variables.